### PR TITLE
Change the application layout in documentation to show packaging by features.

### PR DESCRIPTION
**Pull request by renjithgr**
Fixes gh-9886.

### CODE REVIEW

**Comment by renjithgr** [comment]
Fixed it.

File: spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc [hunk 1]
```diff
@@ -358,18 +358,20 @@ Here is a typical layout:
...
-	         |
-	         +- service
+	         |   +- CustomerController.java
 	         |   +- CustomerService.java
+	         |   +- CustomerRepository.java
 	         |
-	         +- web
-	             +- CustomerController.java
+	         +- order
+	         |   +- Order.java
```
Comment (wilkinsona) [inline spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc]: The `|` isn't needed in this line or in the following three

File: spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc [hunk 2]
```diff
@@ -358,18 +358,20 @@ Here is a typical layout:
 ----
 	com
 	 +- example
-	     +- myproject
+	     +- myApplication
```
Comment (wilkinsona) [inline spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc]: Package names are usually all lower case. Please change this back to `myproject`.

Comment (renjithgr) [inline spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc]: That was silly of me. Fixed it. :)

File: spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc (hunk 1 above)
Comment (renjithgr) [inline spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc]: Fixed this too.

**Comment by wilkinsona** [comment]
Looks good now. Thank you.

**Pull request closed without merging**